timemachine/lib/potentials.py

```diff
@@ -374,10 +391,10 @@ def unbound_impl(self, precision):
 
         custom_ctor = getattr(custom_ops, cls_name_base)
 
-        return custom_ctor(*self.args)
+        return custom_ctor(_kernel_source_dir, *self.args)
```

Comment:
Nice! So the signature of the Python constructors doesn't need to change to accommodate the new `kernel_dir` argument 👍